src/components/views/rooms/wysiwyg_composer/utils/createMessageContent.ts

```diff
@@ -39,6 +42,15 @@ function getHtmlReplyFallback(mxEvent: MatrixEvent): string {
     return (mxReply && mxReply.outerHTML) || "";
 }
 
+function getTextReplyFallback(mxEvent: MatrixEvent): string {
+    const body = mxEvent.getContent().body;
+    const lines = body.split("\n").map(l => l.trim());
```

Comment:
`body` may not be defined. Probably we should check if body is a string here.

Reply:
It's c/c from the `EditMessageComposer` so I'm not sure :)

Reply:
I added a check

Reply:
Better than before, but we expect a string here. Otherwise we may not have a `split` function. We had some trouble with weird event content in the past. That is the reason why I am a bit picky here ;)